rfc6962-bis.xml

```diff
@@ -1147,10 +1147,13 @@ entry specified by <spanx style="verb">start</spanx>.
     </section>
     <section title="TLS Servers" anchor="tls_servers">
       <t>
-        TLS servers MUST use at least one of the three mechanisms listed below to present one or more SCTs or inclusion proofs from one or more logs to each TLS client during TLS handshakes, where each SCT or inclusion proof corresponds to the server certificate or to a name-constrained intermediate the server certificate chains to. Three mechanisms are provided because they have different tradeoffs.
+        TLS servers MUST use at least one of the three mechanisms listed below to present one or more SCTs from one or more logs to each TLS client during TLS handshakes, where each SCT corresponds to the server certificate or to a name-constrained intermediate the server certificate chains to. TLS servers SHOULD present relevant inclusion proofs together with Signed Tree Heads(<spanx style="verb">TransItem</spanx> of types <spanx style="verb">inclusion_proof</spanx> or <spanx style="verb">signed_tree_head</spanx>) in addition to the SCTs. When presented, the inclusion proof MUST be for computing the root in the accompanying Signed Tree Head (presenting inclusion proofs increases client privacy by reducing the inclusion checking to STH consistency checking and reduces load on log servers).
```

Comment:
I suggest: s/relevant/corresponding/

Nit: Need space after "Heads".

Nit: "inclusion_proof" and "signed_tree_head" now have a "_v2" suffix.  (I'm calling this a nit 'cos this text will need to change again anyway once we define that new data structure that will hold SCT + inclusion proof + STH - but let's do that in a separate ticket and separate PR).